src/test/rgw/bucket_notification/api.py

```diff
@@ -257,3 +259,14 @@ def delete_all_topics(conn, tenant, cluster):
             rm_result = admin(['topic', 'rm', '--tenant', tenant, '--topic', topic['name']], cluster)
             print(rm_result)
 
+def change_rgw_config_option(rgw_port, config_option, config_val, **kwargs):
```

Comment:
please move to the test_bn.py file, as this function won't work in general.
so, can't be treated as "test infra"